# HG changeset patch
# User Rafael Laboissiere <rafael@debian.org>
# Date 1202453731 -3600
#      Fri Feb 08 07:55:31 2008 +0100
# Node ID 2a1cb886ec9821b7bc608cf47c6a397b8ed0590c
# Parent  2467639bd8c0a311f4619c6ccfeb790227b156aa
Drop .LO macros from manpages

The .LO are not documented anywhere and processing the manpages through man
(e.g. "man --warnings -l octave.1") yields:

<standard input>:20: warning: `LO' not defined

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,13 @@
+2008-02-08  Rafael Laboissiere  <rafael@debian.org>
+
+	* interpreter/octave.1: Dropped unknow LO macro
+	* interpreter/octave-bug.1: Likewise	
+
 2008-02-07  David Bateman  <dbateman@free.fr>
 
 	* interpreter/sparse.txi: Remove references to spkron.
 
 2008-02-06  John W. Eaton  <jwe@octave.org>
 
 	* interpreter/Makefile.in (stmp-html, HTML): New targets.
 	($(HTML_IMAGES_PNG)): Also depend on stmp-html.
diff --git a/doc/interpreter/octave-bug.1 b/doc/interpreter/octave-bug.1
--- a/doc/interpreter/octave-bug.1
+++ b/doc/interpreter/octave-bug.1
@@ -14,17 +14,16 @@
 .\"
 .\" You should have received a copy of the GNU General Public License
 .\" along with Octave; see the file COPYING.  If not, see
 .\" <http://www.gnu.org/licenses/>.
 .\"
 .\" This page was contributed by Dirk Eddelbuettel <edd@debian.org>
 .\" 
 .TH octave-bug 1 "6 March 2000" GNU
-.LO 1
 .SH NAME
 octave-bug \- report a bug in GNU Octave
 .SH SYNOPSIS
 .B octave-bug
 .RB [\| \-s
 .IR subject \|]
 .SH DESCRIPTION
 .B octave-bug
diff --git a/doc/interpreter/octave.1 b/doc/interpreter/octave.1
--- a/doc/interpreter/octave.1
+++ b/doc/interpreter/octave.1
@@ -12,17 +12,16 @@
 .\" FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
 .\" for more details.
 .\"
 .\" You should have received a copy of the GNU General Public License
 .\" along with Octave; see the file COPYING.  If not, see
 .\" <http://www.gnu.org/licenses/>.
 .\"
 .TH Octave 1 "Jan 8 1996"
-.LO 1
 .SH NAME
 octave \- A high-level interactive language for numerical computations.
 .SH SYNOPSIS
 .nf
 octave [options]
 .fi
 .SH OPTIONS
 The complete set of command-line options for octave is available by
